src/containers/Event.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import {IChangeEvent} from 'react-jsonschema-form';
+import {EditableForm} from '../lib/cms/components/EditableForm';
+import {isUserAuthorized} from '../utils/auth';
+import {ICMSData} from '../models';
+
+interface IEventContext {
```

Comment:
You're right, this isn't related to the event page, but rather to the whole application, and should therefore be named ``ICmsContext``.